baseline-error-prone/src/main/java/com/palantir/baseline/errorprone/StringBuilderConstantParameters.java

```diff
@@ -83,11 +83,11 @@ public Description matchMethodInvocation(MethodInvocationTree tree, VisitorState
                     .build();
         }
         List<ExpressionTree> arguments = result.get();
-        Stream<String> prefixStream = arguments.stream().findFirst()
+        Stream<String> prefixStream = arguments.stream()
+                .findFirst()
                 .map(ASTHelpers::getType)
-                .filter(type ->
-                        ASTHelpers.isSameType(type, state.getTypeFromString("java.lang.String"), state))
-                .map(ignored -> Stream.<String>of())
+                .filter(type -> ASTHelpers.isSameType(type, state.getTypeFromString("java.lang.String"), state))
+                .map(ignored -> Stream.empty())
```

Comment:
does adding a `Stream.<String>empty()` help here?

Reply:
What if we remove the refaster rule in favor of an error-prone rule? They're a lot more precise, and afaict cover all cases.

Reply:
Fixed by #1068